pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/rbac/auth_proxy_role_binding.go

```diff
@@ -43,6 +44,11 @@ func (f *AuthProxyRoleBinding) SetTemplateDefaults() error {
 const proxyRoleBindinggTemplate = `apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
+  labels:
+    app.kubernetes.io/name: proxy-rolebinding
+    app.kubernetes.io/component: auth-proxy
```

Comment:
```suggestion
    app.kubernetes.io/component: rbac
```

Reply:
updated in `kustomize/v1` plugin in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/8a2c1d7c48ec7743828b14a04ae9de765cde30b2

Reply:
Should be finished in both `kustomize` plugins as of commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/daaca98d6775fe54fda2a0017d2f0f4b1369c1de

Marking this particular comment as resolved.